ibu-imager/seedcreator/seedcreator.go

```diff
@@ -129,29 +129,43 @@ func (s *SeedCreator) CreateSeedImage() error {
 }
 
 func (s *SeedCreator) gatherClusterInfo(ctx context.Context) error {
-	// TODO: remove after removing usage of clusterversion.json
-	clusterVersion := &v1.ClusterVersion{}
-	if err := s.client.Get(ctx, types.NamespacedName{Name: "version"}, clusterVersion); err != nil {
-		return err
-	}
-	clusterManifest, err := s.manifestClient.CreateClusterInfo(ctx)
-	if err != nil {
-		return err
-	}
+	if _, err := os.Stat("/var/tmp/gather_cluster_info.done"); os.IsNotExist(err) {
```

Comment:
It would probably be useful to have a common prefix or dir for all these .done files, with maybe a note or an option in the cleanup script to wipe them all if the user wants to do a fresh rerun. If the user is generating multiple seed images, they may want to ensure everything no steps are skipped, since this is primarily to save time for dev testing

Reply:
Maybe define constants for the paths for these files?

Reply:
I think it's a very good idea, will do

Reply:
bonus points if the entire "check if file with Sprintf("%s/%s.done", constDoneFilesPath, name) exists, do something, create file" logic can be handled by a common higher-order-function

Reply:
Javi you can use the common checks folder located in `/var/tmp/checks` (more info in #91)

Reply:
All comments addressed :) please review again ^_^